#the id is used to indicate the record file
string id
---
int8 status